pkg/services/accesscontrol/roles_test.go

```diff
@@ -9,29 +9,41 @@ import (
 )
 
 func TestPredefinedRoles(t *testing.T) {
-	for name, r := range FixedRoles {
+	testFixedRolesMap := func(key, value interface{}) bool {
+		name := key.(string)
+		role := value.(RoleDTO)
+
 		assert.Truef(t,
 			strings.HasPrefix(name, "fixed:"),
 			"expected all fixed roles to be prefixed by 'fixed:', found role '%s'", name,
 		)
-		assert.Equal(t, name, r.Name)
-		assert.NotZero(t, r.Version)
-		// assert.NotEmpty(t, r.Description)
+		assert.Equal(t, name, role.Name)
+		assert.NotZero(t, role.Version)
+		return true
 	}
+	FixedRoles.Range(testFixedRolesMap)
 }
 
 func TestPredefinedRoleGrants(t *testing.T) {
-	for _, v := range FixedRoleGrants {
+	testFixedRoleGrantsMap := func(_, value interface{}) bool {
```

Comment:
This test doesn't make sense anymore, the roles list isn't hardcoded now. However it raises a question, do we want to sort insert roles in the roles list? Is it important?